build.gradle

```diff
@@ -1,31 +1,32 @@
 plugins {
     id 'java'
-    id 'org.jetbrains.kotlin.jvm' version '1.4.21' //Set the Kotlin version here
-    id 'maven'
-    id 'maven-publish'
     id 'idea'
     id 'eclipse'
-    id "edu.wpi.first.GradleRIO" version "2021.2.1"
+    id 'maven-publish'
+    id "edu.wpi.first.GradleRIO" version "2021.3.1"
+    id 'edu.wpi.first.GradleVsCode' version '1.0.0'
 }
 
-group = 'com.github.frc5431'
-
-// sourceCompatibility = JavaVersion.VERSION_11
-// targetCompatibility = JavaVersion.VERSION_11
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+}
 
-repositories {
-    mavenCentral()
+allprojects {
+    repositories {
+        mavenCentral()
+        mavenLocal()
+    }
+    group = 'com.github.frc5431'
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
-    
     implementation wpi.deps.wpilib()
+    implementation wpi.deps.vendor.java()
+
+    // Remove Native Zips with GradleRIO 2022
```

Comment:
does this comment mean to remove the bottom two lines with the next gradlerio version?

Reply:
No, you would remove the following
```gradle
nativeZip wpi.deps.wpilibJni(wpi.platforms.roborio)
nativeDesktopZip wpi.deps.wpilibJni(wpi.platforms.desktop)
nativeZip wpi.deps.vendor.jni(wpi.platforms.roborio)
nativeDesktopZip wpi.deps.vendor.jni(wpi.platforms.desktop)
```